content/waku/2024-gantt.md

```diff
@@ -0,0 +1,103 @@
+        %% Estimate TBC - assuming parallel work possible
+        TBC Review connection management: crit, review-conn-mgmt-2, after storev3-g, 8w
+        Sharding peer mgmt and discovery hardening (go-waku): crit, sh-peer-mgmt-g, after review-conn-mgmt-2, 8w
+    section golang eng 4
+        %% Review if actually needed
+        TBC Store v3 (sync): crit, 2024-02-08, 2024-04-26
+        Reliability Protocol for Resource-Restricted Clients (status-go): crit, rel-reqres-g, 2024-05-13, 10w
+        (metric) Count store messages: count-store-msg, after rel-reqres-g, 2w
+    section golang eng 5
+        %% Estimate TBD - Prem says fine, waiting on 2nd opinion
+        TBC DoS protection for req-res protocols (go-waku client only): crit, dosreqresg, 2024-05-20, 4w
+        (telemetry) direct message reliability: telem-d-msg-rel, 2024-06-17, 6w
+        (telemetry) Measure Bandwidth: telem-bandwidth, after telem-d-msg-rel, 8w
+    section test eng 1 (TBC)
+        %% TBC estimate
+        Peer and connection management tests: sim-conn-mgmt, 2024-05-13, 4w
+        %% TBC estimate
+        (simulation) Functionality and stress test store v3: sim-storev3, after sim-conn-mgmt, 4w
+        %% TBC estimate
+        %% This is Store Service Upgrade - item (2) in DST simulation - start with small scale to get faster results
+        (simulation) Compare store topologies: sim-store-cmp, after sim-storev3, 6w
+        %% TBC estimate
+        (simulation) relay reliability performance impact: sim-relay-rel, after sim-store-cmp sim-req-res rel-relay-g rel-relay-n, 4w
+        %% TBC estimate
+        (simulation) req-res reliability performance impact: sim-reqres-rel, after sim-relay-rel rel-reqres-g, 6w
+    section nim eng 1
+        %% Only dogfooding remaining
+        Store v3-beta (msg hash): crit, storev3-br, 2024-01-01, 2024-05-23
+        Store v3 (sync) research + RFC: crit, storev3-r, 2024-03-25, 14w
+        Investigation/bug fixing of DST results: crit, 2024-07-01, 4w
+    section nim eng 2
+        PostgreSQL Maintenance: crit, pgsql, 2024-01-01, 2024-05-24
+        Reliability Protocol for Relay (nwaku): crit, rel-relay-n, 2024-05-13, 12w
+    section nim eng 3
+        DoS Protection for Req-Res Protocols: crit, dosreqresn, 2024-02-01, 18w
+        Store v3-beta + v3 (dogfooding placeholder):  storev3-df, after storev3-br storev3-r, 4w
+        Store v3-beta + v3 (nwaku hardening): crit, storev3-n, after storev3-df dosreqresn, 3w
+        (telemetry) Fleet logging: telem-fleet-logging, after storev3-n, 4w
+    section nim eng 4
+        %% TBC estimate/end
+        TBC Sharding peer mgmt and discovery hardening (nwaku): crit, sh-peer-mgmt-n, 2024-05-13, 12w
+    section js eng 1
+        Reliability for Req-Res Protocols (light client): crit, rel-reqres-j, 2024-05-01, 12w
+    section js eng 2
+        %% TBC scope
+        TBC Sharding peer mgmt and discovery hardening (light client): crit, sh-peer-mgmt-j, 2024-06-01, 4w
```

Comment:
@weboko @jm-clius please check.
While the JS output is not necessary for Status, I think if we have an JS engineer working on this in js-waku and collaborating with the go engineer that does it in go-waku for light client side, we'll be able to yield higher quality results.

Reply:
looks good to me, this way we will be able to share expertise we have so far and gain new from the context of Status.

Reply:
Agreed.